README.md

```diff
@@ -36,14 +43,21 @@ You need to have [borgbackup](https://github.com/borgbackup/borg/) prior to your
     cd backup-with-borg
     make install
     cp /etc/backup/{example,default}.env
+    cp /etc/backup/{example,default}.exclude
 
     # edit default.env to match backup data
     $EDITOR /etc/backup/default.env
 
+    # edit default.exclude to modify any patterns that should be excluded.
+    # NOTE: Any directories that contain an empty file called `.nobackup` will automatically be excluded from the backup.
+    $EDITOR /etc/backup/default.exclude
+
     # initialize everything
     backup init -e <preferred_encryption_technique>
+    # e.g.: 
+    backup init -e repokey-blake2
 
     # add cronjob (for hourly backup)
-    { crontab -l ; echo "0 * * * * backup"; } | crontab -
+    { crontab -l ; echo "0 * * * * /usr/local/bin/backup"; } | crontab -
```

Comment:
`/usr/local/bin` ... see my comments below.

Reply:
When writing a crontab (especially root's crontab), hard paths should be coded for whatever you are running. Not doing so easily allows an attacker to compromise a system. If two "backup" scripts, for example, are on the path, an attacker could write one that would get executed by root and cron would have no way of reporting that.

Reply:
I see your point. But whenever you've got a path in your PATH variable, which is not strictly limited to be writable by the user itself or root, you've got a problem. A different problem and a problem not limited to cron.

The correct solution for that is not to hardcode a single executable path, **you have to hardcode *every* path!**

Reply:
I updated the README with additional information. I think in this case, providing an example for the crontab entry is OK.